packages/components/config/rollup.config.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable camelcase */
+import nodeResolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import nodeGlobals from 'rollup-plugin-node-globals';
+import { terser } from 'rollup-plugin-terser';
+import postcss from 'rollup-plugin-postcss';
+import json from '@rollup/plugin-json';
+import { dependencies, peerDependencies, name } from '../package.json';
+import { createFilter } from '@rollup/pluginutils';
+import glob from 'glob';
+
+const globMapper = (mapper) => glob.sync(mapper).reduce((acc, item) => {
+    const [ path ] = item.split('/index.js');
+    const last = path.split('/').pop();
+
+    return {
+        ...acc,
+        [last === 'src' ? 'index' : last]: item
+    };
+}, {});
+
+const externalDeps = Object.keys({ ...dependencies, ...peerDependencies }).map(item =>
```

Comment:
Same here, this pattern is probably going to be exactly the same for the rest of the packages